style/style.css

```diff
@@ -291,28 +291,33 @@ html{
     border: none;
     outline: none;
     text-align: left;
-    margin-left: 100px;
+    /* margin-left: 100px; */
      padding: 15px 20px; 
-    font-size: 35px;
+    /* font-size: 35px; */
```

Comment:
Feel free to remove all the excess code instead of commenting it out for the final commit; Git will help us recover the changes if necessary